arch/arm/src/rp23xx/rp23xx_start.c

```diff
@@ -125,21 +140,6 @@ void __start(void)
   rp23xx_lowsetup();
   showprogress('A');
 
-  /* Move the initialized data section from its temporary holding spot in
-   * FLASH into the correct place in SRAM.  The correct place in SRAM is
-   * give by _sdata and _edata.  The temporary location is in FLASH at the
-   * end of all of the other read-only data (.text, .rodata) at _eronly.
-   */
-
-#ifdef CONFIG_BOOT_RUNFROMFLASH
-  for (src = (const uint32_t *)_eronly,
-       dest = (uint32_t *)_sdata; dest < (uint32_t *)_edata;
-      )
-    {
-      *dest++ = *src++;
-    }
-#endif
-
   showprogress('B');
```

Comment:
call showprogress() after arm_earlyserialinit()

Reply:
UART already initialized in rp23xx_losetup(), showing progress is fine